### ISSUE TITLE
config.ini file doesn't reference correct homedir paths by default

### ISSUE THREAD

**Issue by infokiller** [qdore/Mutate#82]
The {home} reference doesn't work in my system, and neither $HOME and other things I tried. It did work however when I searched and replaced it with the home dir path. Is there a way to specify the home directory in this file symbolically?

**Comment by Limero** [comment]
Are you talking about something like: ~/ ?

**Comment by infokiller** [comment]
Yes. I tried both ~ and $HOME and it didn't work for me.

**Comment by mgolisch** [comment]
yeah i think this {home} thing is only meant to be easily replaced by sed or something
like the rpm and deb packaging scripts do it.
Maybe the documentation for installation from sourcecode should mention that.
It just says to copy the config examples over.
Took me a moment to understand why its not working.
